tests/unit/query-utils.test.js

```diff
@@ -10,82 +10,70 @@ const tap = require('tap')
 const cleanQuery = require('../../lib/query-utils')
 
 tap.test('Obfuscate GraphQL query args tests', (t) => {
-  t.test('Should remove new line and tabs', (t) => {
-    const query = `query {
+  t.test('Should obfuscate query args', (t) => {
+    const query = `query logans(run: "(333") {
       runner
     }`
 
-    const newQuery = cleanQuery(query)
+    const argLocations = [
+      {
+        start: 13,
+        end: 24
+      }
+    ]
+
+    const newQuery = cleanQuery(query, argLocations)
 
-    t.equal(newQuery, query)
+    t.notOk(newQuery.includes('333'))
+    t.ok(newQuery.includes('logans(***)'))
 
     t.end()
   })
 
-  t.test('Should hide one arg', (t) => {
-    const query = `query logans(run: 333) {
+  t.test('Should obfuscate aliased query args for aliased', (t) => {
+    const query = `query thing: logans(run: "(333") {
       runner
     }`
 
-    const newQuery = cleanQuery(query)
+    const argLocations = [
+      {
+        start: 20,
+        end: 31
+      }
+    ]
+
+    const newQuery = cleanQuery(query, argLocations)
 
     t.notOk(newQuery.includes('333'))
+    t.ok(newQuery.includes('logans(***)'))
```

Comment:
this old test also asserted the alias is still in tact https://github.com/newrelic/newrelic-node-apollo-server-plugin/pull/124/files#diff-915ccd552c280a0ad5499a8da2e508cf2e4cf4dcc7c981a04c9850bacaac9b57L61